README.md

```diff
@@ -12,11 +12,13 @@ Supported platforms
 pam_hbac was tested on the following operating systems and releases:
   * Linux (RHEL-5 and newer)
     * I tested RHEL-5 and newer Red Hat based distributions. Ubuntu is
-      used as a CI platform, but no functional testing was done there.
+      used as a CI platform, but no functional testing was done there
+      except integration tests
```

Comment:
On Tue, Apr 05, 2016 at 07:08:56AM -0700, Pavel Reichl wrote:
> > @@ -12,11 +12,13 @@ Supported platforms
> >  pam_hbac was tested on the following operating systems and releases:
> >    * Linux (RHEL-5 and newer)
> >      * I tested RHEL-5 and newer Red Hat based distributions. Ubuntu is
> > -      used as a CI platform, but no functional testing was done there.
> > +      used as a CI platform, but no functional testing was done there
> > +      except integration tests
> 
> You are missing dot at the end of sentence, but I can live with that :-)

Thanks, resubmitted.

> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/jhrozek/pam_hbac/pull/62/files/0dc26b27e430ba4c89d898ad8ceb9cec0b59b170#r58544032